src/Microsoft.OData.ModelBuilder/Vocabularies/Capabilities/Constants.cs

```diff
@@ -0,0 +1,212 @@
+            public const string Description = "Description";
+
+            /// <summary>Property LongDescription of Org.OData.Capabilities.V1.InsertRestrictions.</summary>
+            public const string LongDescription = "LongDescription";
+        }
+
+        internal static class UpdateRestrictions
+        {
+            /// <summary>Org.OData.Capabilities.V1.UpdateRestrictions</summary>
+            public const string Term = "Org.OData.Capabilities.V1.UpdateRestrictions";
+
+            /// <summary>Property Updatable of Org.OData.Capabilities.V1.UpdateRestrictions</summary>
+            public const string Updatable = "Updatable";
+
+            /// <summary>Property Upsertable of Org.OData.Capabilities.V1.UpdateRestrictions</summary>
+            public const string Upsertable = "Upsertable";
+
+            /// <summary>Property DeltaUpdateSupported of Org.OData.Capabilities.V1.UpdateRestrictions</summary>
+            public const string DeltaUpdateSupported = "DeltaUpdateSupported";
+
+            /// <summary>Property Permissions of Org.OData.Capabilities.V1.UpdateRestrictions.</summary>
+            public const string Permissions = "Permissions";
+
+            /// <summary>Property Description of Org.OData.Capabilities.V1.UpdateRestrictions.</summary>
+            public const string Description = "Description";
+
+            /// <summary>Property LongDescription of Org.OData.Capabilities.V1.UpdateRestrictions.</summary>
+            public const string LongDescription = "LongDescription";
+        }
+
+        internal static class DeleteRestrictions
+        {
+            /// <summary>Org.OData.Capabilities.V1.DeleteRestrictions</summary>
+            public const string Term = "Org.OData.Capabilities.V1.DeleteRestrictions";
+
+            /// <summary>Property Deletable of Org.OData.Capabilities.V1.DeleteRestrictions</summary>
+            public const string Deletable = "Deletable";
+
+            /// <summary>Property Permissions of Org.OData.Capabilities.V1.DeleteRestrictions.</summary>
+            public const string Permissions = "Permissions";
+
+            /// <summary>Property Description of Org.OData.Capabilities.V1.DeleteRestrictions.</summary>
+            public const string Description = "Description";
+
+            /// <summary>Property LongDescription of Org.OData.Capabilities.V1.DeleteRestrictions.</summary>
+            public const string LongDescription = "LongDescription";
+        }
+
+        internal static class ReadRestrictions
+        {
+            /// <summary>Org.OData.Capabilities.V1.ReadRestrictions</summary>
+            public const string Term = "Org.OData.Capabilities.V1.ReadRestrictions";
+
+            /// <summary>Property ReadByKeyRestrictions of Org.OData.Capabilities.V1.ReadRestrictions.</summary>
+            public const string ReadByKeyRestrictions = "ReadByKeyRestrictions";
+        }
+
+        internal static class ReadRestrictionsBase
```

Comment:
With respect to ease of use, is it better to represent `ReadRestrictionsBase` or ditch it in favour of independent `ReadRestrictions` (and `ReadByKeyRestrictions` if necessary) constants?